lib/Cake/Cache/Engine/FileEngine.php

```diff
@@ -109,7 +109,7 @@ public function gc($expires = null) {
  * @return bool True if the data was successfully cached, false on failure
  */
 	public function write($key, $data, $duration) {
-		if ($data === '' || !$this->_init) {
```

Comment:
This will result in cache files being written for '' content. But I think that is OK and having PHP7 compatibility is more important. We might want to document this in the 2.8 migration guide though.